src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1245,10 +1242,9 @@ public boolean canBeInstantiated() {
 					protected Chunk deserialize(final Fields fields) throws StreamCorruptedException {
 						final World w = fields.getObject("world", World.class);
 						final int x = fields.getPrimitive("x", int.class), z = fields.getPrimitive("z", int.class);
-						Chunk c;
-						if (w == null || (c = w.getChunkAt(x, z)) == null)
+						if (w == null)
 							throw new StreamCorruptedException();
-						return c;
+						return w.getChunkAt(x, z);
```

Comment:
i think getChunkAt may return null in some circumstances, do you know if this is true?

Reply:
I'm not sure... but the docs says that it's ``@NotNull`` as well

Reply:
I did some investigation, according to https://helpch.at/docs/1.9/ although it doesn't say it's ``@NotNull`` (no null notations used at that time), but it never mentions returning null being a possibility, unlike some other functions that do mention null is a possible return value.